config/Configure.help

```diff
@@ -557,6 +557,10 @@ CONFIG_ROMFS_FS
 CONFIG_ROMFS_BASE
   Base of the file system in memory (in paragraphs).
   Typically 0x8000 for a 512K ROM in a 1 MB address space.
+CONFIG_ROMFS_FS_DIRECT_CSEG
+  Saying y to this option runs text segments directly from ROM, as
+  opposed to making copies of them in RAM. Not supported for
+  compressed or relocatable binaries.
```

Comment:
I'm of the belief that having everything "just work" with fewer options is better. Thus, I'd like to use CONFIG_ROMFS_FS as the controlling define for this enhancement. You might want to move this nice descriptive comment over to above the new code in exec.c.

Reply:
That is not a problem to change, but I'm concerned about a situation in which someone, for whichever reason, explicitly relies on the functionality of the text segment technically being writable on a standard ELKS setup, given the lack of memory protection.

Reply:
> explicitly relies on the functionality of the text segment technically being writable on a standard ELKS setup,

The code segment is already shared when possible across processes, so code relying on writing the code segment breaks that assumption and would break when run simultaneously. 

My direction with ELKS is simplicity - we have too few users knowledgeable enough to set all the proper options as it is, and I'm trying to get to a point where there's such a thing as a standard distribution which works very well out of the box for a majority of users. The less options the better - I've spent the last five years removing them and consider that progress.

Reply:
Okay then.

(My personal position is that such issues should be resolved by better configuration wizards, not by removing options, but it has to also be said that fewer options means fewer debugging headaches and reduced code complexity, and it's not my labor I'd be arguing about, so...)